india_compliance/gst_india/doctype/purchase_reconciliation_tool/reco_tool_detail_view.html

```diff
@@ -0,0 +1,75 @@
+<div class="container">
+	<table class="table table-bordered table-hover">
```

Comment:
You can create a map and loop over it. Will make HTML more manageable.
Refer this [link](https://github.com/resilient-tech/india-compliance/blob/f9b753bd0ccdd00b99003b206f73424cca2ef4fe/india_compliance/gst_india/print_format/e_invoice/e_invoice.html#L15).